llvm/lib/Target/RISCV/RISCVSchedule.td

```diff
@@ -211,90 +211,57 @@ def ReadFClass16         : SchedRead;
 def ReadFClass32         : SchedRead;
 def ReadFClass64         : SchedRead;
 
+// For CPUs that support Zfhmin, but not Zfh.
```

Comment:
Just a concern, there may be some risks of misuse.

Reply:
Hopefully tablegen will give an error if you use the same Write* twice in the same scheduler.